src/components/containers/seo.tsx

```diff
@@ -124,7 +137,13 @@ const SEO = ({
             bodyAttributes={{
                 dir: lang_direction,
             }}
-            title={title}
+            title={
+                title
+                    ? typeof title === 'string'
+                        ? localize(title)
+                        : localize(title.key, title.values)
+                    : null
+            }
```

Comment:
title as undefined will no broke application
```suggestion
            title={
                 typeof title === 'string'
                        ? localize(title)
                        : localize(title.key, title.values)
            }
```